spec/models/metric/capture_spec.rb

```diff
@@ -83,4 +77,69 @@
       described_class.perf_capture_health_check(miq_server.zone)
     end
   end
+
+  describe ".perf_capture_now?" do
+    let(:target) { FactoryGirl.build(:host_vmware) }
+
+    before do
+      stub_settings(
+        :performance => {
+          :capture_threshold_with_alerts => {:host => 2},
+          :capture_threshold             => {:host => 10},
+        }
+      )
+    end
+
+    context "realtime host" do
```

Comment:
This is a confusing name because it implies you are doing a realtime capture, which you are not.  I think this should say "with a host with alerts" and the other one should say "with a host without alerts".